src/components/Territory.vue

```diff
@@ -103,14 +104,21 @@ export default {
     ...mapActions({
       getTerritory: 'territory/getTerritory',
       resetNHRecords: 'territory/resetNHRecords',
+      checkinTerritory: 'territory/checkinTerritory',
     }),
 
-    async resetNH() {
-      if (window.confirm('Are you sure you want to reset NH records?')) {
+    async checkIn() {
+      if (window.confirm('Ready to check-in the territory?')) {
         this.isLoading = true;
         await this.resetNHRecords(this.id);
         await this.getTerritory(this.id);
+        await this.checkinTerritory({
+          territoryId: this.id,
+          userId: get(this.territory, this.territory.status.publisher.id),
```

Comment:
the correct lodash get usage expects a string for the second param, like this:

get(territory, 'status.publisher.id')